lib/Controller/RegisterController.php

```diff
@@ -230,8 +233,13 @@ public function showUserForm(string $secret, string $token, string $username = '
 		return new TemplateResponse('registration', 'form/user', [
 			'email' => $registration->getEmail(),
 			'email_is_login' => $this->config->getAppValue('registration', 'email_is_login', 'no') === 'yes',
-			'username' => $username,
+			'loginname' => $loginname,
```

Comment:
Not for now. But maybe the initial state later is better here? Since it is all JS anyways

Reply:
the registration page works totally without JS atm, tried to avoid it to make sure we dont break the webview of any client :-X
Only the admin page is heavily using JS.